flasc/energy_ratio/energy_ratio_visualization.py

```diff
@@ -20,7 +20,7 @@
 from floris import tools as wfct
 
 
-def plot(energy_ratios, labels=None):
+def plot(energy_ratios, labels=None, label_uq=False):
     """This function plots energy ratios against the reference wind
     direction. The plot may or may not include uncertainty bounds,
     depending on the information contained in the provided energy ratio
```

Comment:
Could you update the docstring to include this new option?

Reply:
Done!